src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -1487,6 +1485,50 @@ class OptionsSegment(BaseSegment):
     )
 
 
+class NotEnforcedSegment(BaseSegment):
+    """NOT ENFORCED clause for a primary key."""
+
+    type = "not_enforced_segment"
+    match_grammar = Sequence(
+        "NOT",
+        "ENFORCED",
+    )
```

Comment:
Does this need to be a Segment? Or could it just be a simpler Grammar?

Segments are typically used when you need to reference this (but the `type`) in a rule but I doubt that's the case here.

Or alternatively, since this is so simple, should this just be hardcoded in the two places it's needed rather than being a separate class/grammar?

Reply:
I was just mirroring the PrimaryKey and Foreign Key Segments. These two things together do constitute an expression block.

I'm happy either way

Reply:
Yeah let's remove this one if you don't mind. Seems needless. And there's lots of examples of simple expression blocks being hardcoded.

Reply:
Cool